tests/OmniSharp.Roslyn.CSharp.Tests/TypeLookupFacts.cs

```diff
@@ -452,7 +452,7 @@ public class TestClass
 }";
             var response = await GetTypeLookUpResponse(content);
             var expected =
-            @"DoWork is a method in the TestClass class. System.Console.WriteLine(System.String) for information about output statements.";
+            @"DoWork is a method in the TestClass class. System.Console.WriteLine(System.String)  for information about output statements.";
```

Comment:
why are we adding extra whitespace here?

Reply:
Same as above